.github/ISSUE_TEMPLATE/new-material-request.md

```diff
@@ -0,0 +1,62 @@
+---
+name: New material request
+about: Suggest an idea for something to be added to the docs
+title: ''
+labels: docs, new
+assignees: ''
+
+---
+
+<!--
+  Feel free to delete anything that definitely does not apply to your
+  issue, as well as these comments. (Or don’t, it’s up to you.)
+
+  If your request has to do with an addition to existing material,
+  please paste (a) link(s) from your browser showing the issue (bare
+  so it can be seen, not buried in a Markdown link).
+
+  Please note this is _not_ the place to request a new feature be
+  added to the Perl 6 language--even a small change, like an
+  additional multi variant or a tweak to a method parameter. We
+  only document the language as it is today.
+
+  If you're not sure where your request could be best dealt with,
+  either provide the URL where you think it should be, provide
+  multiple possible URLs, or delete the line. If you just have an idea
+  of the section ("Language," "Types", etc.) include that in "The
+  problem" below.
+
+  Please don't provide a URL you wish existed that does not -- if you
```

Comment:
Remove this. I don't think it's common enough to warrant a warning, and even if a URL is suggested it doesn't hurt anything.

Reply:
> Remove this. I don't think it's common enough to warrant a warning, and even if a URL is suggested it doesn't hurt anything.

It isn’t common, but the point was to reinforce that it’s okay to delete the line entirely when it isn’t applicable and that we’d rather that be done than presenting a URL filled in because it’s a field and people fill fields in. OTOH, issues where people don’t really give us a clue where they’re referring to because they think their English description of the location will be specific enough when it isn’t _is_ common, so the field’s existence is useful.